trucks.js

```diff
@@ -503,6 +504,38 @@ var foodTrucks = [
 	}
 ];
 
+function getTrucks() {
+	return _.map(foodTrucks, function(item) {
+		return item;
+	})
+}
+
+function getTruck(truck) {
+	return _.filter(foodTrucks, function(item) {
```

Comment:
The second argument passed to _.filter should be a function that returns true or false. The way to think about this is: "filter takes a function that informs it whether to include the current item in the filtered list it returns". So it expects a true or false and includes the item accordingly. This means that filter is going to return a list, or rather an array. To find just an individual item, another method, like _.find() would be better suited to the task.